### PR TITLE
Verify that a user is able to create and delete a discourse account

### PR DESCRIPTION
**Pull request by viorelaioia-zz**

### CODE REVIEW

File: pages/account.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+class Account(Base):
+    _delete_account_button_locator = (By.CSS_SELECTOR, '.delete-account button')
+    _delete_account_confirmation_locator = (By.CSS_SELECTOR, '.modal-footer .btn-danger')
+
+    def delete_account(self):
+        self.selenium.find_element(*self._delete_account_button_locator).click()
+        self.wait_for_element_visible(*self._delete_account_confirmation_locator)
+        self.selenium.find_element(*self._delete_account_confirmation_locator).click()
+        from pages.home_page import Home
```
Comment (johngian) [inline pages/account.py]: Why do we import this locally here?

File: pages/base.py [hunk 2]
```diff
@@ -55,3 +55,13 @@ def click_logout_menu_item(self):
         self.click_avatar()
         self.selenium.find_element(*self._logout_button_locator).click()
         WebDriverWait(self.selenium, self.timeout).until(lambda s: self.is_login_button_displayed)
+
+    def click_preferences(self):
+        self.selenium.find_element(*self._preferences_button_locator).click()
+        from pages.account import Account
```
Comment (johngian) [inline pages/base.py:61]: Same here.

File: pages/base.py [hunk 3]
```diff
@@ -55,3 +55,13 @@ def click_logout_menu_item(self):
...
+        return Account(self.base_url, self.selenium)
+
+    def create_new_user(self, email):
+        self.login(email)
+        from pages.register import Register
```
Comment (johngian) [inline pages/base.py:66]: Same here.

File: pages/register.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    _name_locator = (By.ID, 'new-account-name')
+    _create_new_account_button_locator = (By.CSS_SELECTOR, '.modal-footer .btn-primary')
+
+    def enter_username(self, username):
+        self.selenium.find_element(*self._username_locator).clear()
+        self.selenium.find_element(*self._username_locator).send_keys(username)
+
+    def click_create_new_account_button(self):
+        self.selenium.find_element(*self._create_new_account_button_locator).click()
+        from pages.home_page import Home
```
Comment (johngian) [inline pages/register.py]: Same here

**Review by johngian — CHANGES_REQUESTED** [review]

File: pages/account.py (hunk 1 above)
Comment (viorelaioia-zz) [reply pages/account.py]: I added the import here as it is needed in this method only. I can move it to the top though.

Comment (viorelaioia-zz) [reply pages/account.py]: Import moved to the top

File: pages/register.py (hunk 4 above)
Comment (viorelaioia-zz) [reply pages/register.py]: Import moved to the top

**Pull request merged**